src/app/drive/components/ShareDialog/ShareDialog.tsx

```diff
@@ -581,6 +571,7 @@ const ShareDialog = (props: ShareDialogProps): JSX.Element => {
                     <User
                       user={user}
                       key={user.email}
+                      avatarSrc={avatarBlob ? URL.createObjectURL(avatarBlob) : user.avatar}
```

Comment:
the user.avatar is not the same as the one that is passed to the hook?

Reply:
Yes, it is there as a fallback. We check if there is a new avatar to download and save it to the database. In case the custom hook fails, we have `user.avatarBlob` as a fallback, just in case.... Maybe it is still valid.